addons/vehicle_damage/functions/fnc_handleCookoff.sqf

```diff
@@ -22,8 +22,8 @@
  * Public: No
  */
 
-params ["_vehicle", "_chanceOfFire", "_intensity", "_source", "_instigator", ["_hitPart", ""], ["_canRing", true], ["_canJet", true]];
-TRACE_8("handleCookoff",_vehicle,_chanceOfFire,_intensity,_source,_instigator,_hitPart,_canRing,_canJet);
+params ["_vehicle", "_chanceOfFire", "_intensity", "_sourceObj", "_instigator", ["_hitPart", ""], ["_canRing", true], ["_canJet", true]];
```

Comment:
I'd prefer if we kept it `_source`, given both `cookoff` and `vehicle_damage` use that variable to designate the entity who's weapon was to shoot the vehicle.